addons/sourcemod/scripting/scp_sf/items.sp

```diff
@@ -1734,7 +1738,7 @@ public Action Items_FragTimer(Handle timer, int ref)
 			DispatchKeyValue(explosion, "iMagnitude", "500");
 			DispatchKeyValue(explosion, "iRadiusOverride", "350");
 			// don't want particles or sound, we create them seperately
-			DispatchKeyValue(explosion, "spawnflags", "916");
+			DispatchKeyValue(explosion, "spawnflags", "978");
```

Comment:
I know this wasn't there previously, but could you document what exactly these spawnflags do in the comment above?

Reply:
To clarify, something along the lines of:
```
// 1 = sets a thing, 2 = sets another thing
```

Reply:
Would've added [defines from source code](https://github.com/ValveSoftware/source-sdk-2013/blob/0d8dceea4310fde5706b3ce1c70609d72a38efdf/mp/src/game/server/explode.h#L11-L25) and use it together to set spawnflag value.

Reply:
> Would've added [defines from source code](https://github.com/ValveSoftware/source-sdk-2013/blob/0d8dceea4310fde5706b3ce1c70609d72a38efdf/mp/src/game/server/explode.h#L11-L25) and use it together to set spawnflag value.

Did this and moved explosion creation to stocks so everything is in the same place